Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2838,25 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for
+     * the admin screen.
```

Comment:
But both vim and phpstorm provide a way to show the 120th column, maybe you could try that?